docs/src/components/Feedback/index.js

```diff
@@ -1,13 +1,50 @@
-import React from "react";
+import React, { useState } from "react";
 import styles from './styles.module.css';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+
 export const Feedback = () => {
+
+    const { siteConfig } = useDocusaurusContext();
+    const openAnIssue = siteConfig?.customFields?.externalLinks?.openAnIssue;
+    const kubernetesSlackOpenEBS = siteConfig?.customFields?.externalLinks?.kubernetesSlackOpenEBS;
+    const [isThanksTextVisible, setThanksTextVisibility] = useState(false); 
+    
+    const handlePositiveResponse = () => {
+        setThanksTextVisibility(true);
+        sendFeedback("Yes", 1);
+    }
+
+    const handleNegativeResponse = () => {
+        setThanksTextVisibility(true);
+        sendFeedback("No", 0);
+    }
+
+    function sendFeedback(action,value) {
+        if(typeof window !== undefined) {
+            if(!window?.gtag) { 
+                return null;
+            }
+            window?.gtag('event', action, {
+                'event_category': 'Docs:Helpful',
+                'event_label': window.location.href,
+                'value': value
+            });
+        }
+    }
+
     return(
         <div className={styles.wrapper}>
             <h4>Was this page helpful? We appreciate your feedback</h4>
             <div className={styles.buttonGroup}>
-                <button className="doc-button doc-button-primary doc-button-curved">Yes</button>
-                <button className="doc-button doc-button-outlined doc-button-curved">No</button>
+                <button type="button" className="doc-button doc-button-primary doc-button-curved" onClick={() => handlePositiveResponse()} disabled={isThanksTextVisible}>Yes</button>
+                <button type="button" className="doc-button doc-button-outlined doc-button-curved" onClick={() => handleNegativeResponse()} disabled={isThanksTextVisible}>No</button>
```

Comment:
instead of giving seperate functions can we do it with single one `handleResponse('param')`

Reply:
this is also a way. Multiple ways to do it 😄